incountry/crypto_utils.py

```diff
@@ -10,83 +13,91 @@
 
 
 KEYS_TO_HASH = SERVICE_KEYS + SEARCH_KEYS
+EXTRA_KEYS_TO_ENCRYPT = ["precommit_body"]
 
 KEYS_TO_OMIT_ON_ENCRYPTION = ["created_at", "updated_at"]
 
-RESERVED_KEYS_TO_SEND = ["version", "is_encrypted"]
+RESERVED_KEYS_TO_SEND = ["body", "version", "is_encrypted"]
 
 
-def validate_crypto(crypto):
+def validate_crypto(crypto: InCrypto):
```

Comment:
Here you could use `None` as return type:

```suggestion
def validate_crypto(crypto: InCrypto) -> None:
```